tests/answer.test

```diff
@@ -0,0 +1,10 @@
+Authorization failed
```

Comment:
Nit: This is generally referred to as the golden output.

Personally, I'd move it into the C++ file (as a multiline string constant), have some `--golden` flag to print this, and `diff` against this "golden output" from the `Makefile`. So that the things are in one place.

And/or you could even amend to the golden output in the C++ code as each step progresses, and then have the final check that the "collected golden output" equals the "condensed golden output". With the message being "Please update the string constant at the top of this file, to reflect your changes to the logic of the test."

Single source-of-truth for the win, and locality matters for future readers, including yourself 😇